scripts/kfctl.sh

```diff
@@ -92,6 +107,17 @@ createEnv() {
   fi
 }
 
+# For minikube single script download experience
+function download_kfctl_scripts() {
```

Comment:
You can delete this;
We have a separate script to do the download.sh
https://github.com/kubeflow/kubeflow/blob/master/scripts/download.sh